Topic 0/src/point3/HospitalBuilder.java

```diff
@@ -0,0 +1,22 @@
+package point3;
+
+public abstract class HospitalBuilder {
```

Comment:
[High] It is better to have an interface as HospitalBuilder.
This is because Java only allows to extend just for one class (but implements several interfaces, as much as you want). So usually you intend to implement interfaces when this is possible.
In this case, you can't define property *hospital* and you have to move it to each ConcreteBuilder